plugins/outputs/iotdb/iotdb_test.go

```diff
@@ -0,0 +1,377 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package iotdb
+
+import (
+	"fmt"
+	"math"
+	"testing"
+	"time"
+
+	"github.com/apache/iotdb-client-go/client"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/metric"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+var (
+	testHost     = "localhost" // The server's (ip) address that you want to connect to.
+	testPort     = "6667"      // The server's port that you want to connect to.
+	testUser     = "root"
+	testPassword = "root"
+)
+
+func newTestClient() *IoTDB {
+	testClient := newIoTDB()
+	testClient.Host = testHost
+	testClient.Port = testPort
+	testClient.User = testUser
+	testClient.Password = testPassword
+	testClient.Log = testutil.Logger{}
+	return testClient
+}
+
+func TestConnectAndClose(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode")
+	}
+	testClient := newTestClient()
+
+	var err error
+	err = testClient.Connect()
+	require.NoError(t, err)
+	err = testClient.Close()
+	require.NoError(t, err)
```

Comment:
This is easier to read I think
```suggestion
	require.NoError(t, testClient.Init())
	require.NoError(t, testClient.Connect())
	require.NoError(t, testClient.Close())
```

Reply:
You are right! It's shorter and much more beautiful. Thanks. It's fixed.